app/controllers/concerns/partner_filters.rb

```diff
@@ -1,30 +1,37 @@
 # frozen_string_literal: true
 
-class PartnerCategoryFilter
-  def initialize(site, params)
+class PartnerFilters
+  attr_reader :neighbourhoods, :neighbourhood_names
+
+  def initialize(current_site, neighbourhood_names, params)
+    @current_neighbourhood_name = params[:neighbourhood_name]
```

Comment:
is there a reason to use the string of a neighbourhood and not an ID? this seems like a lot of extra work for something that is not in the A/C (?)

Reply:
There absolutely is - https://github.com/geeksforsocialchange/PlaceCal/pull/1999#issuecomment-1729665685 - also the way these names are assigned is way more complicated than you would think, which is why I moved all the name logic into the partner model, and decided to just pass the names themselves

Reply:
But then we just have to work back the logic for how a name is assigned in here, instead of through the name method?